### PR TITLE
added release notes for 10.17.0.364

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<td>Cockpit</td>
+<td>The tooltip in the "Linear gauge" widget no longer overflows the widget card. Moreover, the tooltip color changes according to the type of the declared range (default, red or yellow).</td>
+<td>MTM-55005</td>
+<td>10.17.0.347</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>An issue with module federation has been fixed. If the shell application was built on a windows machine the modules have not been shared properly between shell and plugin.</td>
```
Comment (rssag) [inline content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md:51]:
```suggestion
<td>An issue with module federation has been fixed. If the shell application was built on a Windows machine the modules have not been shared properly between shell and plugin.</td>

```

File: content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Administration</td>
+<td>Microservices which have been created via API without providing a binary for it can again be subscribed without getting an error message.</td>
```
Comment (rssag) [inline content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md:34]: is "can again be subscribed" correct? it looks like there's a "to" missing, but i'm not sure if that was the intended meaning.

File: content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<td>MTM-56037</td>
+<td>10.17.0.347</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>Resolved the issue that versioned applications might not be unpacked upon core startup, which resulted in a HTTP 404 error for some requests.</td>
```
Comment (rssag) [inline content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md:42]:
```suggestion
<td>Resolved the issue that versioned applications might not be unpacked upon core startup, which resulted in an HTTP 404 error for some requests.</td>

```

File: content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>In rare cases, when the real-time connection was interrupted, the UI did not resubscribe to the previously subscribed channels once the connection was re-established. This behavior has been fixed.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md:35]: @reey Please see comments from Richard above.

File: content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md (hunk 2 above)
Comment (BeateRixen) [reply content/release-10-17-0/platform-10-17-0-bundle/platform-10-17-0-364.md:34]: We often have these to versions: "A microservice/application is subscribed", but "a tenant subscribes to a microservice/application". So Id say it is correct here.

File: content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (reey) [reply content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md:35]: So the consequence was that e.g. measurements of certain widgets did not update again after the connection was reestablished..
The UI is re-establishing a connection and via this connection it is then resubscribing..

[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md:35]: Thanks @reey; I rephrased it a bit to make it clearer to users. Can you please approve that this is correct?

[the remark this answers is not in the document]
Comment (reey) [reply content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-364.md:35]: It was not actually limited to widgets, but should be fine :)

**Pull request merged**